cmd/composectl/cmd/update/complete.go

```diff
@@ -0,0 +1,49 @@
+package updatectl
+
+import (
+	v1 "github.com/foundriesio/composeapp/pkg/compose/v1"
+	"github.com/foundriesio/composeapp/pkg/update"
+	"github.com/spf13/cobra"
+)
+
+var (
+	completeCmd = &cobra.Command{
```

Comment:
I don't see why you are making all these command structs "global" variables. You can keep these isolated inside the `init` function and also define the `.Run` in the same declaration.

Reply:
There is no particular reason for that. I just thought that having them private within the `updatectl` package makes them isolated enough.
I can move them into the `init` if you think there is benefit in it. However IMHO, the code looks better structured if command definition is at the top of a file in the `var` section.

Reply:
I prefer keeping the structs inside `init`, where they are actually used, avoiding the need to look at two separate places when following the code. This seems, in fact, to be the most usual practice.